### ISSUE TITLE
ask for creation WVGA monochrome mt9v034 json config 

### ISSUE THREAD

**Issue by habib-Boloorchi** [ArduCAM/ArduCAM_USB_Camera_Shield#83]
Hi, 
I have seen that everyone uses mt9v034 with this resolution: 750 * 480 in monochrome. I need this for my project. would you pls create this configuration. do I need a need a new sdk for it? I am using it for my raspberry pi in ROS and I am not kind of new to hardware and configuration. would you give me some instruction about its confiig file too?
Thank you,

**Comment by ArduCAM** [comment]
Would you please let me know what is the exact problem you have right now？

**Comment by habib-Boloorchi** [comment]
I saw in the data sheet that we can get 750*480 monochrom(grayscale) image from mt9v034. But when I change the format to 4 or change the size to 750*480 .), it does not show any thing in rqt_image_show.
I use this camera with raspberry pi and get data by trogger node of ros.

**Comment by ArduCAM** [comment]
Can you run the default setting without any modification?
Simply modify the config file might not be able to work properly.

**Comment by habib-Boloorchi** [comment]
yes I can run default one it is the image of the results andits rate:
![image](https://user-images.githubusercontent.com/43946196/49243516-2c332b00-f3d3-11e8-8477-1436940d0580.png)
I want to get grayscale data because I am using this image for specific project. can you help me with this?
